policy/modules/contrib/certmonger.te

```diff
@@ -124,8 +124,12 @@ optional_policy(`
 ')
 
 optional_policy(`
-    cockpit_manage_pid_dirs(certmonger_t)
-    cockpit_manage_pid_files(certmonger_t)
+	gen_require(`
```

Comment:
Does this approach make sense?

Reply:
Or maybe should I move these into cockpit.te in cockpit repo?

Similar to https://github.com/cockpit-project/cockpit/pull/16223

Reply:
@zpytela @vmojzis WDYT?